### DISCUSSION TITLE
Calling a webhook for notifications of actions

### DISCUSSION THREAD

**Discussion by Jacob-Tate** [gotson/komga#1637] [Ideas]
It would be nice if we could have komga call a webhook when specific events occur such as when a new issue or manga is added, when an issue / chapter is finished, and other various events. This would be great for tracking my progress on another app and for knowing when new issues are added via a discord webhook.

**Comment by gotson** [comment]
You can subscribe to SSE. There's no plans to add webhook support.

**Reply by Jacob-Tate** [reply]
What is SSE in this context?

**Reply by gotson** [reply]
[server sent events](https://en.wikipedia.org/wiki/Server-sent_events)
